### DISCUSSION TITLE
Screenshots

### DISCUSSION THREAD

**Discussion by heavyrain266** [Nooo37/bling#51] [Show and tell]
### Show us your awesome setup made with bling and love.

**Rules**
- Please use reactions instead of writting comments like "Nice setup!".
- Try posting images directly to github attaching them to comments.
- If you want to share your dotfiles, post them along with image.
- **Please star Bling and follow Nooo37 and JavaCafe01 for their great work!**

**Comment by Nooo37** [comment]
Since people often don't use the possibility of changing the master counter, they forget about it but it is extremely usefull in the mstab layout (probably even more so than in other layouts). So here a quick reminder that it exists.

![](https://user-images.githubusercontent.com/70270606/101285449-7e3e9080-37e5-11eb-9776-3c2d737e9d9d.png)

**Comment by heavyrain266** [comment]
Tilled wallpaper module is pretty nice
![screeenshot](https://user-images.githubusercontent.com/71175207/101392749-055f3780-38be-11eb-8df9-a875a6981106.png)

**Comment by MCotocel** [comment]
https://user-images.githubusercontent.com/74227209/115113998-13287600-9f9e-11eb-88a0-30c422e76e4d.mov

Scratchpads with Awestore animations are useful and look great

Edit: My [dotfiles](https://github.com/MCotocel/linux-dotfiles/tree/unstable)

**Comment by davidnoronha1** [comment]
![S](https://user-images.githubusercontent.com/38278035/115380268-069b5c00-a1f0-11eb-99c3-10afc4d8dc68.png)
Pretty basic use ~ ive only used the tabbed tag preview and playerctl module
the tabbed thing is hidden ;-;

**Reply by chxp82q** [reply]
@undefinedDarkness Nice setup! I can see some transparent artifacts in the corners from the tag preview module. This is because the widget base itself is transparent to support AA rounding with AwesomeWM, and this shows when using shadows. There are a couple of solutions to this:

1.  You can enable true shadows.

```conf
wintypes:
{
    normal = {full-shadow = true;};
};
```
This allows `picom` to ignore the transparent artifacts from the corners, and still apply shadows with AA from AwesomeWM.

2. Round with picom instead.

The official git master branch supports AA rounding on legacy backends.

3. Turn off shadows for the specific widget.
4. Change the radius and don't round.

**Comment by Piterden** [comment]
![2021-05-20_20-20](https://user-images.githubusercontent.com/5930429/119022198-05a14a00-b9a9-11eb-99e6-d64f0f1f283d.png)
Index with yellow background shows parent tag number
Also I make theme switcher based on rofi, but don't know how to make a shot with rofi launched

**Reply by chxp82q** [reply]
@Piterden If you want to get a screenshot with `rofi`, you can use `sleep` as a countdown timer to open `rofi`.

**Reply by Piterden** [reply]
Oh, thank you, I will try

**Comment by JezerM** [comment]
A simple, based on Gruvbox, theme.
I also made a kind of dock on the right, with task preview~

![Screenshot_2021-07-27-42_1366x768](https://user-images.githubusercontent.com/59768785/127199701-e1d0f328-3e34-435d-963c-44f0f70e8ae8.png)

**Reply by Nooo37** [reply]
Epic! Are you setting your titlebars background based on the window?

**Reply by JezerM** [reply]
Nah, the titlebar color is always the same, `#282828`.
Left window is just with lower opacity.
But that's a great idea!

**Reply by heavyrain266** [reply]
too much opacity on that window and bar which kills whole theme imo.
